proto/regen/ecocredit/v1/types.proto

```diff
@@ -62,3 +62,18 @@ message OriginTx {
   // the id of a transaction based on a type (tx id, serial number)
   string id = 2;
 }
+
+// CreditTypeProposal is a gov Content type for adding a credit type.
+message CreditTypeProposal {
```

Comment:
Maybe this should go in a alpha/beta package so we can remove it with 0.46

Reply:
Moving forward, I think we should avoid using `alpha` if we are planning to include it on mainnet. I also think we should avoid using multiple versions of the same package within the same release but maybe this would be an exception if a separate package is necessary.

If we create a separate package, we would need to name the file something different if we want to use the same destination i.e. `"github.com/regen-network/regen-ledger/x/ecocredit/core"`. If we want to use a different destination, we could create a `v2beta1` subdirectory.

We will need to remove `Params` as well when we update to Cosmos SDK v0.46. I think creating a separate package might be more trouble than its worth.

Reply:
If we don't create a separate package then we should deprecate those in the next release. Is that an acceptable solution?

Reply:
That sounds good to me.